addons/l10n_ee/data/template/account.tax-ee.csv

```diff
@@ -203,13 +203,13 @@
 "","","","","","","","","","","tax","invoice","","","",""
 "","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","tax","refund","","","",""
-"l10n_ee_vat_in_0_kms_41_1","370","0% KMS §41¹","purchase","20.0","percent","0%","tax_group_vat_20","False","12","base","invoice","+1_1||+7_1","","","0% KMS §41¹"
+"l10n_ee_vat_in_0_kms_41_1","370","20% KMS §41¹","purchase","20.0","percent","0%","tax_group_vat_20","False","12","base","invoice","+1_1||+7_1","","","20% KMS §41¹"
```

Comment:
Maybe change the description so it is consistent with the invoice one ? 
`20% Special`

Same for the 22% one below